packages/server/lib/controllers/oauth.controller.ts

```diff
@@ -1545,6 +1545,9 @@ class OAuthController {
                 );
             } else {
                 await updateSuccessActivityLog(activityLogId, template.auth_mode === 'CUSTOM' ? null : true);
+                if (template.auth_mode === 'CUSTOM') {
```

Comment:
why only when `CUSTOM`

Reply:
this seems incorrect indeed unless I missed something

Reply:
Yup, thanks, removed